
Files already downloaded and verified
/usr/local/lib/python3.7/dist-packages/torch/utils/data/dataloader.py:481: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 2, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  cpuset_checked))
Training:   0% 0/50000 [00:15<?, ?it/s, loss=1.3863]





Evaluating:  93% 365/391 [00:08<00:00, 43.11it/s]





























































































Training:   0% 0/50000 [03:48<?, ?it/s, loss=1.3863]
Traceback (most recent call last):
  File "starter.py", line 25, in <module>
    main(config)
  File "starter.py", line 10, in main
    launcher(config)
  File "/content/drive/MyDrive/Collab_starter/pytorch-gan-collections/launch.py", line 258, in launcher
    train(config)
  File "/content/drive/MyDrive/Collab_starter/pytorch-gan-collections/launch.py", line 241, in train
    f.write(f'Step: {step}, FID: {FID:.4f}, IS: {IS:.4f}\n')
TypeError: unsupported format string passed to tuple.__format__